os/include/tinyara/pm/pm.h

```diff
@@ -284,15 +288,17 @@ enum pm_state_e {
 								 *
 								 * PM_SLEEP may be following by PM_NORMAL
 								 */
+	PM_COUNT,
 };
 
 /* This structure contain pointers callback functions in the driver.  These
  * callback functions can be used to provide power management information
  * to the driver.
  */
 
-struct pm_callback_s {
-	struct sq_entry_s entry;	/* Supports a singly linked list */
+struct pm_callback_s
```

Comment:
Please check the violations of coding rules. This file is not in arch so that you must fix the violation.

Reply:
ok, will update